### DISCUSSION TITLE
I am trying to add a custom domain to my Pages site, but I am seeing a blank page and no clues as to why

### DISCUSSION THREAD

**Discussion by MadeehaKhan** [community/community#145735] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

[Here is my repo](https://github.com/MadeehaKhan/Madeeha-Khan-Website). I followed the instructions on how to set up my DNS settings with the A records, the CNAME record, as well as the TXT record with the secret (that one is under show more):
![image](https://github.com/user-attachments/assets/890f88d2-8513-4848-b3d4-aee900d7d2f2)

I have deployed to my custom domain, and am seeing a blank page when I open it even after a success message from GitHub. I am now however also seeing this error: 
![image](https://github.com/user-attachments/assets/dc34f447-7134-4f33-a67f-e31f857f83b0)

Please help!

**Comment by ELHart05** [comment, marked as answer]
The issues with your website is that the [CSS](https://madeehakhan.dev/Madeeha-Khan-Website/assets/index-CrsdDi4p.css) and [JS](https://madeehakhan.dev/Madeeha-Khan-Website/assets/index-DhzY8e3b.js) references are not accessible and that's because of the `vite.config.js` base key that's adding a sub route to the files locations making it inaccessible,

![image](https://github.com/user-attachments/assets/accbc5ed-c8c0-46a3-b21d-5929b302f336)
```js
export default defineConfig({
    build: {
        sourcemap: true,
        rollupOptions: {
          onwarn(warning, defaultHandler) {
            if (warning.code === 'SOURCEMAP_ERROR') {
              return
            }
    
            defaultHandler(warning)
          },
        },
      },
    plugins: [react()],
    //base: "/Madeeha-Khan-Website" //REMOVE THIS LINE
})
```
If you still facing this issue I'd suggest to use [vercel](https://vercel.com/) it's better, as it's more suitable for this kind of projects, as github pages are more likely made for simple static websites (you can use github pages but vercel is better), go to [vercel](https://vercel.com/) website, create new project, link your github repo and select vite then click deploy, for the dns mapping you search on how vercel handles the DNS mapping and all good!

**Reply by MadeehaKhan** [reply]
That fixed it, thanks! I'll try out vercel if it gives me any more trouble or for my next project, thank you so much for your help :)

**Reply by ChrisQuashie** [reply]
would you be able to help out with mine, I have a similar issue
